test/e2e/app-dir/third-parties/basic.test.ts

```diff
@@ -43,7 +43,7 @@ describe('@next/third-parties basic usage', () => {
       '[src^="https://www.googletagmanager.com/gtm.js?id=GTM-XYZ"]'
     )
 
-    expect(gtmScript.length).toBe(1)
+    expect(gtmScript.length).toBe(2)
```

Comment:
I updated the test but want to know if this is the correct change.

It might make sense to remove the second script and try to include `gtmAuth` and `gtmPreview` in the javascript.

Reply:
This script shouldn't be duplicated, it looks like it was fine without updating to have the GTM script be injected via the `dangerouslySetInnerHTML` as we already handle it outside of there with the nonce.